stdlib/string.gr

```diff
@@ -55,19 +74,23 @@ export let rec byteLength = (s: String) => {
   ret
 }
 
-
-// @disableGC-safe wrapper
+// disableGC-safe wrapper
 @disableGC
 let wasmSafeByteLength = (s: String) => {
   Memory.incRef(WasmI32.fromGrain(byteLength))
   Memory.incRef(WasmI32.fromGrain(s))
   byteLength(s)
 }
 
-// Find the start index of a substring.
-// @param sub: String - The substring to find
-// @param input: String - The string to check
-// @returns Option<Number>
+/**
+ * Find the start index of a substring.
+ * 
+ * @param sub: The substring to find
+ * @param input: The string to check
+ * @returns The starting index of he substring
+ * 
+ * @example String.indexOf("world", "Hello world")
```

Comment:
```suggestion
 * Finds the start index of a substring in the input string.
 * 
 * @param search: The substring to find
 * @param string: The string to inspect
 * @returns `Some(index)` containing the starting index of the substring if found or `None` otherwise
 * 
 * @example String.indexOf("world", "Hello world") == Some(6)
```
When a function returns an Option (or a Result) type, we write the `@returns` in the form of
> \`Some(descriptiveVariableName)\` containing the *description_of_variable_name* if found (for Option) /successful (for Result) or \`None\` otherwise

I've provided you a sample for Some/None of this function, but you can also look into other docs in the stdlib to see Result or other Option examples.